pkg/core/bundle_unpack.go

```diff
@@ -466,6 +465,7 @@ func downloadBundleEntries(ctx context.Context, bundle *Bundle,
 		bundle.l.Info("downloading bundle entries",
 			zap.Int("num", len(bundle.BundleEntries)))
 		for _, b := range bundle.BundleEntries {
+			var selectionPredicateOk bool
```

Comment:
at first tests were failing b/c i didn't notice that `z` matches `leafsize` 0_o .. made this change before realizing that i was debugging a "derp."

so this is for readability only, then.

Reply:
:+1: